lib/std/c.zig

```diff
@@ -8723,7 +8723,7 @@ pub const NOTE = switch (native_os) {
         pub const EXIT_DETAIL = 0x02000000;
         /// mask for signal & exit status
         pub const PDATAMASK = 0x000fffff;
-        pub const PCTRLMASK = (~PDATAMASK);
+        pub const PCTRLMASK = ~@as(u32, PDATAMASK);
         pub const EXIT_DETAIL_MASK = 0x00070000;
```

Comment:
From Apple's `event.h`:

```c
#define	NOTE_PCTRLMASK	0xf0000000		/* mask for hint bits */
#define	NOTE_PDATAMASK	0x000fffff		/* mask for pid */
```

I don't understand what the original intent was here. Likewise for the FreeBSD prong below.